pootle/static/js/shared/models/project.js

```diff
@@ -46,6 +42,10 @@ export const Project = Backbone.Model.extend({
     return l(['', 'projects', this.get('code'), 'admin', 'permissions', ''].join('/'));
   },
 
+  getFSUrl() {
+    return l(['', 'admin', 'projects', this.get('code'), 'fs', ''].join('/'));
```

Comment:
This makes more sense to be `return l(['', 'projects', this.get('code'), 'admin', 'fs', ''].join('/'));` so the URL is `/projects/testfs/admin/fs/` instead of `/admin/projects/testfs/fs/`.